web/handler/track_test.go

```diff
@@ -0,0 +1,150 @@
+package handler
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+	"net/http/httptest"
+	"net/url"
+	"testing"
+
+	"github.com/camphor-/relaym-server/domain/entity"
+	"github.com/camphor-/relaym-server/domain/mock_spotify"
+	"github.com/camphor-/relaym-server/usecase"
+
+	"github.com/golang/mock/gomock"
+	"github.com/google/go-cmp/cmp"
+	"github.com/labstack/echo/v4"
+)
+
+func TestTrackHandler_SearchTracks(t *testing.T) {
+	albumImageJSONs := make([]*albumImageJSON, 1)
+	albumImageJSONs[0] = &albumImageJSON{
+		URL:    "https://i.scdn.co/image/ab67616d0000b273b48630d6efcebca2596120c4",
+		Height: 640,
+		Width:  640,
+	}
```

Comment:
勿論この書き方でもいいんだけど、 `albumImageJSON`の長さが固定なら一気に書いてしまっても良いかな。
型の変換の時に`make()` でスライスの長さを指定していたのは、普通にappendをすると、スライスの長さが2^xになるたびにメモリのアロケートが走ってしまうから。

https://golang.hateblo.jp/entry/golang-slice-array
https://qiita.com/imoty/items/bb18fb50d526474d2d10
(記事の「[]*T ではなく []Tを使う」に書いてあるとおり、プリミティブ型じゃなくて構造体のポインタを使っていると言うほど効果はないんだけど、癖で毎回makeで作ってしまう）


```suggestion
	albumImageJSONs := []*albumImageJSON{
		{
			URL:    "https://i.scdn.co/image/ab67616d0000b273b48630d6efcebca2596120c4",
			Height: 640,
			Width:  640,
		},
	}
```